cgp/population.py

```diff
@@ -13,24 +13,24 @@ class Population:
 
     def __init__(
         self,
-        n_parents: int,
-        seed: int,
-        genome_params: Union[dict, List[dict]],
+        n_parents: int = 4,
+        seed: int = 1234,
+        genome_params: Optional[Union[dict, List[dict]]] = None,
         individual_init: Optional[Callable[[IndividualBase], IndividualBase]] = None,
     ) -> None:
         """Init function.
 
         Parameters
         ----------
-        n_parents : int
-            Number of parent individuals.
-        seed : int
-            Seed for internal random number generator.
+        n_parents : int, optional
+            Number of parent individuals. Defaults to 4.
+        seed : int, optional
+            Seed for internal random number generator. Defaults to 1234.
         genome_params : dict
-            Parameters for the genomes of the population's individuals.
+            Parameters for the genomes of the population's individuals. Defaults to None.
```

Comment:
Hmm might it be more user-friendly to have the defaults here instead of nested in `Genome()`?

Reply:
At least the defaults should be documented here as well

Reply:
i avoided documenting the results here but put a reference to where they are defined. if we duplicate this information one part will become outdated quickly.